src/vs/workbench/test/common/editor/untitledTextEditor.test.ts

```diff
@@ -303,24 +303,24 @@ suite('Workbench untitled text editors', () => {
 		model.dispose();
 	});
 
-	test('service#onDidChangeLabel', async () => {
-		const service = accessor.untitledTextEditorService;
-		const input = service.create();
+	// test('service#onDidChangeLabel', async () => { // {{SQL CARBON EDIT}} - Disable as untitledTextEditorModel test is disabled.
```

Comment:
I'd prefer just doing test.skip instead - reduces the number of changes were making to vs code stuff